tensorflow_datasets/core/download/download_manager.py

```diff
@@ -238,13 +238,17 @@ def _download(self, resource):
     if isinstance(resource, six.string_types):
       resource = resource_lib.Resource(url=resource)
     url = resource.url
+    download_type = ""
```

Comment:
Can you add a comment explaining what this variable is and what it contains ?

Reply:
Sure, this variable contains the type of data that is being downloaded when a kaggle link is detected.(The suffix that comes denoting the kind of kaggle data we are working on)

Since other links don't need to be processed this way, they don't need the suffix. So, it's kept blank so that it doesn't do any changes to the url when joining the link to the suffix on line 266. If it's a kaggle url, (as seen on the next line, this `download_type` variable contains the suffix.